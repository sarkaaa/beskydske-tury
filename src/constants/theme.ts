import * as colors from "./colors";

const theme = {
  colors
};

export default theme;