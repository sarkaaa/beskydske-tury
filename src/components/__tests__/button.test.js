import React from "react"
import renderer from "react-test-renderer"
import { ThemeProvider } from "styled-components"
import "jest-styled-components"

import Button from "../button"
import theme from "../../constants/theme"

const Wrapper = ({ children }) => (
  <ThemeProvider theme={theme}>{children}</ThemeProvider>
)

test("renders correctly", () => {
  const tree = renderer
    .create(
      <Wrapper>
        <Button onClick={() => {}}>Hello World</Button>
      </Wrapper>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
