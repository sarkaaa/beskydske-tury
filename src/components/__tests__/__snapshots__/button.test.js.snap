// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
.c0 {
  border-radius: 3rem;
  padding: 0.6rem 2.5rem;
  font-size: 1rem;
  font-family: 'Noto Sans',sans-serif;
  color: #FCFCFC;
  border: solid 2px transparent;
  background-image: linear-gradient(rgba(255,255,255,0),rgba(255,255,255,0)),linear-gradient(-50deg,#0D7419,#4ec752,#6ADD6E);
  background-origin: border-box;
  background-clip: content-box,border-box;
  -webkit-transition: linear 0.2s;
  transition: linear 0.2s;
}

.c0:hover {
  cursor: pointer;
  box-shadow: none;
  color: #FCFCFC;
  background-image: linear-gradient(rgba(255,255,255,0),rgba(255,255,255,0)),linear-gradient(-50deg,#0a5913,#5cc460);
}

<button
  className="c0"
  onClick={[Function]}
>
  Hello World
</button>
`;
